x/csr/types/errors.go

```diff
@@ -16,4 +16,7 @@ var (
 	ErrSmartContractSupply = sdkerrors.Register(ModuleName, 1000, "csr::CSRPool")
 	ErrNFTSupply           = sdkerrors.Register(ModuleName, 1001, "csr::CSRPool")
 	ErrMisMatchedNFTSupply = sdkerrors.Register(ModuleName, 1002, "csr::CSRPool")
+	ErrInvalidParams       = sdkerrors.Register(ModuleName, 1003, "csr::Params")
```

Comment:
In the future, I wonder if we should suffix the errors like so, moduleName::Type: overview of error

Reply:
that is what i am doing